apps/L1E/instagram/instagram-frontend/src/components/SearchResult.tsx

```
@@ -0,0 +1,47 @@
+        onChange={(e) => setSearchName(e.target.value)}
+        placeholder="Search username..."
+        className="border p-2 rounded mb-4"
+      />
+
+      {loading && <p>Loading...</p>}
+      {error && <p className="text-red-500">Error: {error.message}</p>}
+
+      {!loading && !error && data?.getProfiles?.length ? (
+        <ul>
+          {data.getProfiles.map((user) => (
+           <a href={`/${user.userName}`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)